lib/oelite/baker.py

```diff
@@ -250,7 +250,7 @@ def show(self):
 
         #meta.dump(pretty=False, nohash=False, flags=True,
         #          ignore_flags=re.compile("filename|lineno"),
-        meta.dump(pretty=True, nohash=(not self.options.nohash),
+        meta.dump(pretty=True, show_nohash=(not self.options.nohash),
```

Comment:
i don't really want to diverge your work, so maybe this should just be ignored since the change is ok as long as you accept the general quality of the current code, but

Methods with boolean args is a CodeSmell, and it is a strong indication that this should be refactored into something else interely. Why not a seperate method dumpVerbose() perhaps?